target/linux/ath79/nand/base-files/etc/init.d/bootcount

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh /etc/rc.common
+
+# SPDX-License-Identifier: GPL-2.0
+
+START=99
+
+start() {
+	. /lib/functions.sh
```

Comment:
Interesting. This is missing in normal ath79 bootcount. So, either functions.sh is sourced somewhere else or this isn't working there.

I'd personally prefer the include outside of start()/boot(), but that's a matter of taste.

Reply:
I just checked and didn't find `functions.sh` in any of the other `bootcount` files. 

Proper operation confirmed.

Reply:
It seems to be sourced in /etc/rc.common

I wonder whether it wouldn't be more correct to source it here, too, but that's out of the scope for this PR.

Thanks for checking BTW.